### PR TITLE
grimshot - added option to 'save-copy' to save to file and copy to clipboard

### PR DESCRIPTION
**Pull request by bugrasan**
as the subject says, added a new option 'save-copy'.
also added a vim modeline to the file.
please check `grimshot.1`, since it seems that my version of `scdoc` deleted some `\&` characters in to output file.

### CODE REVIEW

File: contrib/grimshot.1.scd [hunk 1]
```diff
@@ -23,6 +23,10 @@ grimshot - a helper for screenshots within sway
 *copy*
 	Copy the screenshot data (as image/png) into the clipboard.
 
+*save-copy*
+	Save the screeshot into a regular file (see above for save) and
```
Comment (Xyene) [inline contrib/grimshot.1.scd:27]: - s/screeshot/screenshot
- mark save as `*save*` and copy as `*copy_*, since they refer to commands

**Review by Xyene — CHANGES_REQUESTED** [review]
Thanks -- LGTM, apart from a small nit in the documentation.

File: contrib/grimshot.1.scd (hunk 1 above)
Comment (bugrasan) [reply contrib/grimshot.1.scd]: - misspelling corrected
- marked *save* and *copy* => hope my understanding was correct

**Comment by Ashe** [comment]
Has this just been abandoned? I'd really love this feature :( grimshot is really cool.

**Comment by WhyNotHugo** [comment]
I like the idea, but probably using `--save` and `--copy` would be best (where either or both can be specified). I suggest waiting for #5947 to be merged and base the implementation on the changes done there.

**Comment by emersion** [comment]
The `contrib/` directory has been moved from Sway to a separate repository: https://github.com/OctopusET/sway-contrib

See https://github.com/swaywm/sway/pull/7541 for details.

Please re-open your pull request against the new repository.

**Pull request closed without merging**